packages/react-instantsearch/src/components/InfiniteHits.js

```diff
@@ -14,25 +16,31 @@ class InfiniteHits extends Component {
       hasMore,
       refine,
       translate,
+      header,
+      footer,
     } = this.props;
     const renderedHits = hits.map(hit => (
       <ItemComponent key={hit.objectID} hit={hit} />
     ));
     const loadMoreButton = hasMore ? (
-      <button {...cx('loadMore')} onClick={() => refine()}>
+      <button {...cx(['loadMore'])} onClick={() => refine()}>
```

Comment:
We can avoid the condition, when a `button` is disabled you can't click on it. We can also inline the `button` inside the `return`.